dcos/subcommand.py

```diff
@@ -118,6 +124,30 @@ def distributions():
         return []
 
 
+def command_info(command):
+    """top level documentation of default DCOS-CLI command
+    :param command: name of command
+    :param type: str
+    :returns: command summary
+    :rtype: str
+    """
+
+    return command_documentation(command).split('\n')[0]
```

Comment:
We added `subcommand.default_config_schema`. We should do the same here and add `default_command_info` and `default_command_documentation`. I think the implementation should use `SubcommandMain` what do you think? This will keep things consistent.